integration/simple_app_test.go

```diff
@@ -416,5 +416,82 @@ func testSimple(t *testing.T, context spec.G, it spec.S) {
 				))
 			})
 		})
+
+		context("BP_DISABLE_SBOM is set to true", func() {
+			var (
+				container1 occam.Container
+				container2 occam.Container
+			)
+
+			it.After(func() {
+				Expect(docker.Container.Remove.Execute(container1.ID)).To(Succeed())
+				Expect(docker.Container.Remove.Execute(container2.ID)).To(Succeed())
+			})
+
+			it("skips SBOM generation", func() {
+				var err error
+
+				source, err = occam.Source(filepath.Join("testdata", "simple_app"))
+				Expect(err).ToNot(HaveOccurred())
+
+				var logs fmt.Stringer
+				image, logs, err = pack.WithNoColor().Build.
+					WithPullPolicy("never").
+					WithBuildpacks(
+						settings.Buildpacks.NodeEngine.Online,
+						settings.Buildpacks.BuildPlan.Online,
+					).
+					WithEnv(map[string]string{
+						"BP_LOG_LEVEL":    "DEBUG",
+						"BP_DISABLE_SBOM": "true",
+					}).
+					Execute(name, source)
+				Expect(err).ToNot(HaveOccurred(), logs.String)
+
+				Expect(logs).To(ContainLines("    Skipping SBOM generation for Node Engine"))
+
+				// Ensure node is installed correctly
+				container1, err = docker.Container.Run.
+					WithCommand("echo NODE_ENV=$NODE_ENV && node server.js").
+					WithPublish("8080").
+					Execute(image.ID)
+				Expect(err).NotTo(HaveOccurred())
+
+				Eventually(container1).Should(BeAvailable())
+
+				response, err := http.Get(fmt.Sprintf("http://localhost:%s", container1.HostPort("8080")))
+				Expect(err).NotTo(HaveOccurred())
+				Expect(response.StatusCode).To(Equal(http.StatusOK))
+
+				content, err := io.ReadAll(response.Body)
+				Expect(err).NotTo(HaveOccurred())
+				Expect(string(content)).To(ContainSubstring("hello world"))
+
+				Eventually(func() string {
+					cLogs, err := docker.Container.Logs.Execute(container1.ID)
+					Expect(err).NotTo(HaveOccurred())
+					return cLogs.String()
+				}).Should(
+					ContainSubstring("NODE_ENV=production"),
+				)
+
+				Expect("/layers/sbom").ToNot(BeADirectory())
```

Comment:
Isn't this just checking that the `/layers/sbom` path on your local machine filesystem doesn't exist?

Reply:
Pffffftttt... yes @ryanmoran, that's my bad for rushing 🙃
How about [this](https://github.com/paketo-buildpacks/node-engine/pull/472/files#diff-a864a1d24641f3d716c9b563043e4eb4f208754c3172b6527c03d5802d0d30f4R506)?

Reply:
I think that works. You could also do something like what we currently have for checking that the SBOM does exist: https://github.com/paketo-buildpacks/node-engine/blob/6d1a66e1ad07c48a124fc024da9b232b67715dcf/integration/simple_app_test.go#L166-L169

You'd obviously check that these files don't exist when the SBOM is exported out of the build container.

Let me know what you think.

Reply:
@ryanmoran I was able to use your suggestion by updating the pack version used in the integration test to v0.26.0. We should be good to go now.